assignments/closure.js

```diff
@@ -1,14 +1,42 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+{
+  // This is a closure.
+  // I don't mean to be sassy but this is as simple as it gets.
+  // Not very useful, though.
+}
+
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
```

Comment:
I loved reading how you processed and learned through this problem. Good job finding your answer